DEVELOPMENT.md

```diff
@@ -118,6 +118,20 @@ kubectl create clusterrolebinding cluster-admin-binding \
   --user="${USER}"
 ```
 
+### Install in custom namespace
```

Comment:
i have one completely minor piece of feedback: usually we have an extra newline after a header.

i'm also happy to `lgtm` and let @mattmoor-sockpuppet fix that tho XD

Reply:
cool - thanks, I just added the extra newline anyways.